src/Contract/Crypto.purs

```diff
@@ -0,0 +1,44 @@
+module Contract.Crypto
+  ( hashData
+  , hashScript
+  , datumHash
+  , plutusHash
+  , HashedData(..)
+  , HashMethod(..)
+  ) where
+
+import Contract.Monad (Contract, wrapContract)
+import Data.Either (Either)
+import Data.Maybe (Maybe)
+import Prelude ((<<<))
```

Comment:
We usually import `Prelude` without explicit imports, and place it above every other import separated by a blank line